### PR TITLE
Replace all .com to .org — Changing all .com to .org — DO NOT MERGE UNLESS IN MAINTENANCE WINDOW

### PR DESCRIPTION
**Pull request by callieburuchara**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ VoteAmerica provides a handful of products for other organizations to use:
 * [Civic Data API](api/index.md): 80+ rigorously fact-checked data points per state, covering everything from voter ID laws to voter registration deadlines.
 
 The software is free to use and embed on your website. If you'd like access to the data that comes in via a dedicated instance of our software, you'll need to sign up for a VoteAmerica+ account.  You'll also need to sign up for a VoteAmerica+ account if you'd like to use the Civic Data API.
-- [Click here for the embed code for the free tools](https://www.voteamerica.com/embeds/)
+- [Click here for the embed code for the free tools](https://www.voteamerica.org/embeds/)
```
Comment (ebehlmann) [inline docs/index.md:15]: While we're here ... we've moved this page. It redirects, but we might as well update the link to go directly where we want it to go, which is: https://secure.voteamerica.org/free-software

File: docs/software/index.md [hunk 2]
```diff
@@ -13,10 +13,10 @@ The software has minimal branding, and the fonts and color scheme should work wi
 
 ## Free software
 
-VoteAmerica's software is free for anyone to use. You can find the code to embed it on your website on the [VoteAmerica website](https://www.voteamerica.com/free-software/).
+VoteAmerica's software is free for anyone to use. You can find the code to embed it on your website on the [VoteAmerica website](https://www.voteamerica.org/free-software/).
```
Comment (ebehlmann) [inline docs/software/index.md:16]: Also this one - let's link to https://secure.voteamerica.com/free-software/

**Review by ebehlmann — COMMENTED** [review]
Looks good! Left a couple notes about URLs we may as well update while we're at it.

File: docs/index.md (hunk 1 above)
Comment (callieburuchara) [reply docs/index.md:15]: Done, thank you!

File: docs/software/index.md (hunk 2 above)
Comment (callieburuchara) [reply docs/software/index.md:16]: Done, thank you!

**Review by ebehlmann — APPROVED** [review]

**Pull request merged**